src/cpp/enclave/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(OpaqueEnclaveTrusted)
+
+set(SOURCES
+  crypto/ks_crypto.cpp
+  crypto/m_crypto.cpp
```

Comment:
It's a bit unclear how these two crypto files are divided up -- why are there two crypto files? Maybe we should merge them into one?

Reply:
It might not be worth it to spend effort merging crypto code rn since I'm planning to create the Crypto/Attestation shared repo in the near future.

Reply:
m_crypto.cpp is a modified file from OE as well -- I think we should keep them separate for easier version control as well in case we need to update it.